src/electionguard_db/mongo-init.js

```diff
@@ -1,2 +1,4 @@
 db.createCollection("guardians");
 db.createCollection("key_ceremonies");
+db.createCollection("key_ceremony_deltas", { capped: true, size: 100000 });
```

Comment:
this table holds just a list of the changed key ceremonies, but because it's capped it allows the use of [tailable cursors](https://www.mongodb.com/docs/manual/core/tailable-cursors/)